geonode/maps/views.py

```diff
@@ -785,6 +786,14 @@ def perm_filter(layer):
     }))
 
 
+def map_download_qlr(request):
+    """
+    This function serves as geoserver compatibility.
+    We might need to use QLR on geoserver in the future.
+    """
+    return HttpResponseBadRequest('Sorry, QLR extension has not supported yet')
```

Comment:
IMHO, this will never be done. Because it means on the same server, users will need Geoserver for their main OGC server and also QGIS Server only to generate the QLR.

I think we should totally hide the link if it's not QGIS Server backend and remove the option for Geonode Geoserver user to click on a QLR link.

Reply:
Is there not the same kind of links for `Maps` like for layers ? Like this section : http://staging.geonode.kartoza.com/maps/117/metadata_detail 'References'

Reply:
afaik, there options to choose between geoserver or qgis-server backend. 
this code is to anticipate the scenario although we never use geoserver.

I didn't see the use of Links on the previous implementation, not even in the template.
that's why i don't use them.
do you think we should use them?